src/entry-points/local-file-player/App.svelte

```diff
@@ -65,6 +64,25 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
     videoEl.focus();
   }
 
+  async function onDeleteFile(ind: number) {
+    files = files.filter((file,fileIndex) => fileIndex !== ind)
+    if (currFileInd === ind){
+      if (ind === files.length){
+        // Removed the last file, play the new last file
+        currFileInd = files.length-1
+      }else if (ind === 0){
+        // Removed the first file, play the new first file
+        currFileInd = 0
```

Comment:
I don't think this block makes sense, because it's only executed when `currFileInd` is already `0`.

Reply:
Yes, I will trim it in next commit